mysql/datadog_checks/mysql/activity.py

```diff
@@ -117,7 +117,7 @@ def __init__(self, check, config, connection_args):
             enabled=is_affirmative(config.activity_config.get("enabled", True)),
             expected_db_exceptions=(pymysql.err.OperationalError, pymysql.err.InternalError),
             min_collection_interval=config.min_collection_interval,
-            config_host=config.host,
+            config_host=check.resolved_hostname,
```

Comment:
We shouldn't be calling `check.resolved_hostname` in the constructor as there's a risk that it blows up and prevents the check from even starting. 
 
Do we even need to set `config_host` anymore? 

Same comment for the other async jobs.

Reply:
You're right, it doesn't look like it's being used anywhere anymore. Same goes for the other integration's we own.